constraintFunctions.f90

```diff
@@ -149,6 +149,9 @@ subroutine getEnvVarsAtT( t )
     character(len=maxEnvVarNameLength) :: this_env_var_name, orderedEnvVarNames(size( envVarNames ))
     logical :: pressure_set, rh_set, temp_set
 
+    ! TODO, is this the best place to call calcTheta?
```

Comment:
If only a single member (or only a few) is used, then `only` is useful to ensure other bits aren't accidentally used (part of what's known as defensive programming). That's the usual criteria - if most or all members are used then just use the whole module without `only`. However, in some of the functions I've also used `only` as a useful way of documenting where the different variables come from - for example in `readPhotolysisRates()` and `readPhotolysisConstants()` I've added a long list to the line `use photolysisRates_mod, only : ...` because that's helpful to show where most of the variables are coming from - if it's not explicitly stated then it can be a bit opaque. It also has the added benefit of defensive programming as above, but that's less of the focus.

If it's fairly obvious, and not too much risk of inadvertently using member variables incorrectly, there's no need to add `only`.